src/main.js

```diff
@@ -15,22 +21,35 @@ var currentCover;
 // Add your event listeners here 👇
 showNewRandomCoverButton.addEventListener('click', getRandomCover);
 
+makeNewButton.addEventListener('click', makeCoverButton);
+
 window.onload = (event) => {
   currentCover = createCover(
-    getRandomCover(),
```

Comment:
There was an error on the page and changing getRandomCover to getRandomImage resolved it.

Reply:
ooh great catch, I see why and I can see why it might've still been working for us before! 😅 
Thanks so much for the comment on this change!